### PR TITLE
Small typo fixes

### PR DESCRIPTION
**Pull request by graubs**

### CODE REVIEW

File: index-pt-br.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
 </style>
 </head>
 <body>
-<h1>Código de conduta da conferência</h1>
+<h1>Código de Conduta da Conferência</h1>
```
Comment (MattiSG) [inline index-pt-br.html]: The official rule for titles in Portuguese is lowercase. Capitalising nouns is English. 🙂

[Reference](https://pt.wikipedia.org/wiki/Wikipédia:Votações/Regra_para_t%C3%ADtulos#Proposta_1:_Norma_da_ABNT/Regra_do_acordo_ortográfico_de_1990).

Comment (graubs) [reply index-pt-br.html]: Hi, Matt! I'm aware of that constraint about title captalisation but also there is an rule that says proper name must be capitalised noums. If we are naming the conduct code as 'Código de Conduta da Conferência' then i guess it should be Capitalised. What are your thoughts about this? Thanks!

Comment (graubs) [reply index-pt-br.html]: Matt, as this discussion is not settled yet I'll revert those changes and send only the typo fixes. It's more important change this than the title one. We can discuss the title matter later. Thanks!

Comment (MattiSG) [reply index-pt-br.html]: 👌 
Thanks a lot for this!

**Pull request merged**